lib/fluent/plugin/gcloud_pubsub/client.rb

```diff
@@ -27,10 +32,35 @@ def bytesize
     end
 
     class Publisher
-      def initialize(project, key, autocreate_topic)
+      def initialize(project, key, autocreate_topic, metric_prefix)
         @pubsub = Google::Cloud::Pubsub.new project_id: project, credentials: key
         @autocreate_topic = autocreate_topic
         @topics = {}
+
+        @compression_ratio =
```

Comment:
I don't think this is capturing the compression ratio. See comment where this is calculated.